### ISSUE TITLE
off-canvas navigation not keyboard accessible?

### ISSUE THREAD

**Issue by frankfarm** [n33/skel#115]
Thanks for making Skel available. It's pretty wonderful. One small issue I have is that the off-canvas navigation does not seem to be keyboard accessible. Is there a way to access it via the keyboard and I'm just missing it? e.g., if I visit
http://getskel.com/demos/offcanvasnav.html
then press Tab I expect some kind of visual change to the menu icon then if I press Enter I expect the menu to be activated and permit me to continue navigating among the menu items using Tab, Shift+Tab, and Enter to select a menu item. Is this possible?

**Comment by n33** [comment]
Yep, you're right. Mind re-opening this over at https://github.com/n33/skel-layers though (since this is a Layers plugin issue)?

**Comment by frankfarm** [comment]
Done. Many thanks!
